lib/web_apis/api_importer.es6.js

```diff
@@ -127,8 +145,13 @@ foam.CLASS({
                 interfaceName,
                 apiName,
               });
-              this.interfaceDAO.put(webInterface);
-              webInterface.browsers.put(browser);
+              if (this.toCloudDatastore) {
```

Comment:
Let's make a `RateLimitedDAO` that owns the `RateLimiter`, then whatever creates an `ApiImporter` can inject a `RateLimitedDAO({ ... delegate: DatastoreDAO({ ... }) })`.